packages/fuels-abigen-macro/Cargo.toml

```diff
@@ -20,8 +20,8 @@ syn = "1.0.12"
 
 [dev-dependencies]
 ctor = " 0.1"
-fuel-core = { version = "0.8", default-features = false }
-fuel-gql-client = { version = "0.8", default-features = false }
+fuel-core = { version = "0.9.1", default-features = false }
```

Comment:
You should use major.minor instead of major.minor.patch for Fuel deps.

Reply:
Done!